app/uk/gov/hmrc/tradergoodsprofilesrouter/service/RouterService.scala

```diff
@@ -287,24 +394,94 @@ class RouterServiceImpl @Inject() (eisConnector: EISConnector, uuidService: Uuid
     rawDetail match {
       case regex(code, _) =>
         code match {
-          case "006" =>
+          case ApplicationConstants.InvalidOrMissingEoriCode                                 =>
```

Comment:
Quite a pedantic one, but if you add this to the imports: 
`import uk.gov.hmrc.tradergoodsprofilesrouter.utils.ApplicationConstants._` 

You can shorten remove all the `ApplicationConstants.xxxx` in this code